Bottom: aff826758adfa3e8482a1059fde63343f3944455
Top:    f9ee30199c37f36180fee0d127f9bedd86aacd45
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-15 18:39:38 +0000

find.h: Annotate which exported functions are not used!


---

diff --git a/src/find.h b/src/find.h
index 9644727..7afe520 100644
--- a/src/find.h
+++ b/src/find.h
@@ -47,7 +47,7 @@ bool LineLineIntersect (LineType *, LineType *);
 bool LineArcIntersect (LineType *, ArcType *);
 bool PinLineIntersect (PinType *, LineType *);
 bool LinePadIntersect (LineType *, PadType *);
-bool ArcPadIntersect (ArcType *, PadType *);
+/* NOT USED EXTERNALLY */// bool ArcPadIntersect (ArcType *, PadType *);
 void LookupElementConnections (ElementType *, FILE *);
 void LookupConnectionsToAllElements (FILE *);
 void LookupConnection (Coord, Coord, bool, Coord, int, bool AndRats);
